KarhooUISDK/Screens/CheckoutScreen/Checkout/DetailsCellView.swift

```diff
@@ -61,13 +61,14 @@ struct DetailsCellView: View {
                 .fill(KarhooUI.colors.background2.getColor())
         )
         .onTapGesture(perform: {
-            delegate?.onTap()
+            delegate?.onTap() // will be removed when Old Checkout will be removed
+            viewModel.onTap()
         })
     }
 }
 
 struct DetailsViewCellPreviews: PreviewProvider {
     static var previews: some View {
-        DetailsCellView(model: DetailsCellViewModel(title: "Passanger", subtitle: "add details", iconName: "kh_uisdk_passenger"), delegate: nil)
+        DetailsCellView(viewModel: DetailsCellViewModel(title: "Passanger", subtitle: "add details, this is excample of text containing more than 2, or even more than 3 lines. We need that kind of text for comments cell, in this case input for user is unlimited. ", iconName: "kh_uisdk_passenger", onTap: {print("tapped")}), delegate: nil)
```

Comment:
It's more common to insert a `Lorem ipsum` text in the future to mimic long texts. Can leave this as is, just a note on future long placeholder needs.